src/model/Column.ts

```diff
@@ -139,11 +141,15 @@ export default class Column extends AEventDispatcher {
    * @private
    */
   private compressed = false;
+  renderername: string;
+  protected rendererList: {type: string, label: string}[];
 
   constructor(id: string, public desc: IColumnDesc) {
     super();
     this.uid = fixCSS(id);
     this.cssClass = (<any>this.desc).cssClass || '';
+    this.renderername = (<any>this.desc).type || 'heatmapcustom';
```

Comment:
`desc.type` is part of the `IColumnDesc` so there is no need to cast to any + it is given, so no need for a default value. 

In general, I would introduce a new optional `IColumnDesc` field `rendererType` that defines the default renderer to use. default to `desc.type`. Since this field is valid for all columns it can be part of the `IColumnDesc`